en_US/deploy/cluster/rebalancing.md

```diff
@@ -36,27 +36,31 @@ You can use the following CLI command to start the node evacuation. The `--evacu
 
 ```bash
 ./bin/emqx_ctl rebalance start --evacuation \
+    [--wait-health-check Secs] \
     [--redirect-to "Host1:Port1 Host2:Port2 ..."] \
     [--conn-evict-rate CountPerSec] \
     [--migrate-to "node1@host1 node2@host2 ..."] \
     [--wait-takeover Secs] \
     [--sess-evict-rate CountPerSec]
 ```
 
-| Parameter           | Type             | Description                                                  |
-| ------------------- | ---------------- | ------------------------------------------------------------ |
-| `--redirect-to`     | String           | The redirected server address during reconnection, for MQTT 5.0 clients; Refer to [MQTT 5.0 Specification - Server redirection](https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901255) for more details. |
-| `--conn-evict-rate` | Positive integer | Client disconnection rate, count/second; 500 connections per second by default |
-| `--migrate-to`      | String           | Space or comma-separated list of nodes to which sessions will be evacuated |
-| `--wait-takeover`   | Positive integer | Amount of time in seconds to wait before starting session evacuation; Unit: second, 60 seconds by default |
-| `--sess-evict-rate` | Positive integer | Client evacuation rate, count/second; 500 sessions per second by default |
+| Parameter              | Type             | Description                                                  |
+| -----------------------| ---------------- | ------------------------------------------------------------ |
+| `--wait-health-check`  | Positive integer | Specified waiting time (in seconds, default 60 seconds) for the LB to remove the source node from the active backend node list. Once the specified waiting time is exceeded, the evacuation will start and the node will start rejecting new connections. |
```

Comment:
```suggestion
| `--wait-health-check`  | Positive integer | The duration (in seconds, 60 seconds by default) during which the Load Balancer (LB) waits for the source node to be removed from the active backend node list. When this specified waiting time elapses, the evacuation process begins, and the node starts rejecting any new incoming connections. |
```

Reply:
>Load Balancer (LB) waits for the source node to be removed from the active backend node list
Seems that the situation is reverse 😅 
It is the node who waits: it starts to report unhealthy status and waits. During the wait time, LB is supposed to remove the node from the active list. It is the LB who removes the node.

Reply:
Thanks you! That's exactly the information I want to know! Then I think the description can be changed to "The duration (in seconds, 60 seconds by default) during which the node waits for the Load Balancer (LB) to remove it from the active backend node list. When this specified waiting time elapses, the evacuation process begins, and the source node starts rejecting any new incoming connections."？